src/lib/Pjax/PjaxResponseMainContentMapper.php

```diff
@@ -50,14 +62,46 @@ public function map($response)
         $xpath = new \DOMXPath($doc);
         $title = $xpath->query('//div[@data-name="title"]')[0]->nodeValue;
         $content = $this->innerHtml($xpath->query('//div[@data-name="html"]')[0]);
+        $notificationElements = $xpath->query('//ul[@data-name="notification"]/li');
 
         $this->app->setConfig([
             'title' => $title,
             'toolbars' => ['discovery' => 1],
             'mainContent' => ['result' => '<ez-server-side-content>' . $content . '</ez-server--side-content>'],
+            'notifications' => $this->buildNotifications($notificationElements),
         ]);
     }
 
+    /**
+     * Builds an array of notifications for Hybrid Platform UI app from PJAX
+     * Notification HTML elements.
+     *
+     * @param \DOMNodeList $elements
+     * @return array
+     */
+    private function buildNotifications(\DOMNodeList $elements)
+    {
+        $notifications = [];
+
+        foreach ($elements as $element) {
```

Comment:
it could be done this way but on the other hand, the notifications are really a part of the PJAX response so it also makes sense to do that in the PjaxResponseMainContentMapper. But the parsing becomes more complicated and this class grows (that's unlikely), then yeah it would be a good refactoring.